pkg/sampleprovider/machine_server.go

```diff
@@ -121,7 +128,7 @@ func (ms *MachinePlugin) GetMachineStatus(ctx context.Context, req *cmi.GetMachi
 //
 func (ms *MachinePlugin) ListMachines(ctx context.Context, req *cmi.ListMachinesRequest) (*cmi.ListMachinesResponse, error) {
 	// Log messages to track start of request
-	glog.V(2).Infof("List machines request has been recieved for %q", req.ProviderSpec)
+	glog.V(2).Infof("List machines request has been received")
```

Comment:
Do you think we shouldn't print the `providerSpec` here? I ask because even I am not sure about it.

Reply:
It produces quite a lot of logging. What do you think about logging it only at a verbose level of 4 (see additional commit https://github.com/gardener/machine-controller-manager-provider-sampleprovider/pull/15/commits/f72c6b8fd09c5586112f12fabdd2202117ffe439)?

Reply:
Yes, I think this should be better.